### DISCUSSION TITLE
GEN24_Ladesteuerung v0.12.4

### DISCUSSION THREAD

**Discussion by wiggal** [wiggal/GEN24_Ladesteuerung#47] [Announcements]
Änderungen in html/funktion_Diagram.php
-  Batterieladung vom Netz im Liniendiagramm darstellen
-  FIX: Falscher Eigenverbrauch bei Balkendiagramm Verbrauch

html/funktion_Diagram.php umbenannt in html/7_funktion_Diagram.php

NEU: 
html/8_tab_Diagram.php
html/8_funktion_Diagram.php

Diagramm Aufbereitung nach EnergieQuelle und EnergieZiel,
um Laden aus dem Netz und Einspeisen aus Batterie besser abbilden zu können.

<hr /><em>This discussion was created from the release <a href='https://github.com/wiggal/GEN24_Ladesteuerung/releases/tag/v0.12.4'> GEN24_Ladesteuerung v0.12.4</a>.</em>

**Comment by jaydee73** [comment]
Moin! Gerade installiert und ich würde gerne zwei Anmerkungen anbringen:

Du weist ja immer dankenswerterweise darauf hin, wenn die config.ini bei einem Update "betroffen" ist, sprich auch individuell angepasst werden muss. Das gilt allerdings auch für die config.php, sofern dort von dir Änderungen vorgenommen worden sind. Explizit hier für die Docker-Variante, da man die php da ja normalerweise reinmapped. In der php ist ja z. B. auch der individuelle PV-Wert hinterlegt. Insofern muss der ja eh immer angepasst werden. Bei 12.4 wurde jetzt noch eine achte Kachel hinzugefügt, insofern auch wieder eine geändert config.php.

Und noch ne (optische) Kleinigkeit, die mehr unter die Rubrik Korinthenkackerei fällt: Die Rubrik-Namen "Diagramm" und insbesondere Diagramm2 finde ich nicht so richtig aussagekräftig. ;-) Im Solarweb heißt das "Energiebilanz". Könnte man das hier nicht auch verwenden? Also vielleicht dann "Energiebilanz Quelle" und "Energiebilanz Ziel"? Oder war die Aufsplittung des Diagramms anders gemeint?

**Reply by wiggal** [reply]
Danke für die Hinweise:

- den Hinweis auf die Änderung in der html/config.php hab ich leider in der 0.12.4 vergessen, werd versuchen zukünftig daran zu denken.
- Die Benennung der Diagrammregister sind wirklich überdenkenswert. 
  Energiebilanz ist schon mal ein guter Ansatz, sollte halt nicht zu lang werden.
  Zur Erläuterung:
  
  Diagramm: ist eine Unterscheidung der Energie nach PV-Produktion (wo geht sie hin) und Verbrauch (wo kommt sie her).  Hier fehlen halt immer Daten, wenn aus dem Netz Energie in den Akku bzw. aus dem Akku Energie ins Netz fließt wie im Solarweb.
  
  Diagramm2: hier habe ich versucht nach Energie-Quelle (Netz und PV) bzw. Energie-Ziel (Verbrauch, ins Netz, in den Akku) zu unterscheiden. 

Dazu bräuchten wir nun kurze, treffende Bezeichnungen.

**Reply by wiggal** [reply]
Übrigens kannst du die Reiter über die html/config.php für dich benennen wie du willst.
Einfach den Namen im Array  Feld zwei (z.B. Diagramm ) ändern.

**Reply by wiggal** [reply]
die Registernamen habe ich jetzt mal wie folgt geändert:

  Diagramm  -->> PV-Bilanz = Produktions- und Vergrauchsbilanz
  Diagramm2 -->> QZ-Bilanz = Quellen- und Zielbilanz

Was hältst du davon?

**Reply by jaydee73** [reply]
Klingt gut. Wobei bei "PV- Bilanz" das PV auch gerne mal als Photovoltaik angesehen werden könnte. Aber gut, irgendwas ist ja immer... ;-) Mir fällt auch nichts besseres ein.

Allerdings noch ein kleiner Typo: Es heißt Ver**b**rauchsbilanz, nicht Ver**g**rauchsbilanz.

**Reply by wiggal** [reply]
> Klingt gut. Wobei bei "PV- Bilanz" das PV auch gerne mal als Photovoltaik angesehen werden könnte. Aber gut, irgendwas ist ja immer... 

Das es auch als Photovoltaikbilanz gesehen werden kann ist gewollt, ist ja auch richtig ;-)

> 
> Allerdings noch ein kleiner Typo: Es heißt Ver**b**rauchsbilanz, nicht Ver**g**rauchsbilanz.

Danke, Tippfehler in CHANGELOG.md beseitigt.
